src/server/plugins/router.ts

```diff
@@ -35,9 +46,21 @@ export default {
         },
         {
           method: 'get',
-          path: '/help/privacy',
-          handler(_request: Request, h: ResponseToolkit) {
-            return h.view('help/privacy')
+          path: '/help/privacy/{slug}',
+          handler(
+            request: Request<{ Params: { slug: string } }>,
+            h: ResponseToolkit
+          ) {
+            const { slug } = request.params
+
+            const privacyPolicy = privacyPolicies[slug]
+
+            if (!privacyPolicy) {
+              logger.error(`Privacy policy not found for slug ${slug}`)
+              throw Boom.notFound()
```

Comment:
FYI - if you prefer, in route handlers you can `return Boom.notFound()` rather than `throw`ing it.